packages/cli/src/controllers/role.controller.ts

```diff
@@ -7,7 +17,30 @@ export class RoleController {
 	constructor(private readonly roleService: RoleService) {}
 
 	@Get('/')
-	getAllRoles() {
-		return this.roleService.getAllRoles();
+	async getAllRoles(): Promise<RoleDTO[]> {
```

Comment:
if we change the existing route to get roles, we need to update the front on the same PR